10주차/BOJ_10988_팰린드롬인지확인하기_디노.py

```diff
@@ -0,0 +1,7 @@
+l = input()
+for i in range(len(l)):
+    if l[i] != l[len(l) - 1 - i]:
+        print(0)
+        exit()
+
+print(1)
```

Comment:
최적이 아니라면 .. 이 방법도 .. 
```suggestion
print(int(word == word[::-1]))
```